downstream/modules/platform/con-about-eda-controller.adoc

```diff
@@ -0,0 +1,11 @@
+[id="about-event-driven-ansible-controller_{context}"]
+
+= Event-Driven Ansible controller
+
+[role="_abstract"]
+The Event-Driven Ansible controller is the interface for event-driven automation and introduces automated resolution of IT requests. This component helps you connect to sources of events and acts on those events using rulebooks. This technology improves IT speed and agility, and enables consistency and resilience. With Event-Driven Ansible, you can: 
```

Comment:
Minor grammar suggestion to change: 
This component helps you connect to sources of events and acts
to
This component helps you connect to sources of events and act

Reply:
I had to read this a few times. I think I considered "helps you connect to sources..." and "acts on those events..." as two parallel parts of the sentence, but I can also see the other way as well. Updated accordingly.